<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Filesystem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="loginStatusLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="FSlistView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="gotoParentCatalog_Button">
      <property name="text">
       <string>Go to parent catalog</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pasteButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Paste</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionAdd_user"/>
    <addaction name="actionDelete_user"/>
    <addaction name="actionLogout"/>
    <addaction name="separator"/>
    <addaction name="actionSave_FileSystem"/>
    <addaction name="actionLoad_FileSystem"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="actionsMenu">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <widget class="QMenu" name="menuFile">
     <property name="title">
      <string>File</string>
     </property>
     <addaction name="actionInformation"/>
     <addaction name="actionType"/>
     <addaction name="actionView_privileges"/>
     <addaction name="actionChange_Access"/>
    </widget>
    <addaction name="menuFile"/>
    <addaction name="actionCatalog"/>
   </widget>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="actionAdd_File_2"/>
    <addaction name="actionAdd_Catalog_2"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="actionsMenu"/>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exitAction">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="text">
    <string>Insert Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="removeRowAction">
   <property name="text">
    <string>Remove Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="insertColumnAction">
   <property name="text">
    <string>Insert Column</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, C</string>
   </property>
  </action>
  <action name="removeColumnAction">
   <property name="text">
    <string>Remove Column</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, C</string>
   </property>
  </action>
  <action name="insertChildAction">
   <property name="text">
    <string>Insert Child</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionInfo_about_filesystem">
   <property name="text">
    <string>info about filesystem</string>
   </property>
  </action>
  <action name="actionCheck_latest_version">
   <property name="text">
    <string>check latest version</string>
   </property>
  </action>
  <action name="actionAdd_Catalog">
   <property name="text">
    <string>Add Catalog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionAdd_File">
   <property name="text">
    <string>Add File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionEdit_File">
   <property name="text">
    <string>Edit File</string>
   </property>
  </action>
  <action name="actionEdit_Catalog">
   <property name="text">
    <string>Edit Catalog</string>
   </property>
  </action>
  <action name="actionEdit_Catalog_2">
   <property name="text">
    <string>Edit Catalog</string>
   </property>
  </action>
  <action name="actionEdit_File_2">
   <property name="text">
    <string>Edit File</string>
   </property>
  </action>
  <action name="actionAdd_File_2">
   <property name="text">
    <string>Add File</string>
   </property>
  </action>
  <action name="actionAdd_Catalog_2">
   <property name="text">
    <string>Add Catalog</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionCatalog">
   <property name="text">
    <string>Catalog</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>Information</string>
   </property>
  </action>
  <action name="actionType">
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionView_privileges">
   <property name="text">
    <string>View Permissions</string>
   </property>
  </action>
  <action name="actionChange_Access">
   <property name="text">
    <string>Change Permissions</string>
   </property>
  </action>
  <action name="actionSave_FileSystem">
   <property name="text">
    <string>Save FileSystem</string>
   </property>
  </action>
  <action name="actionLoad_FileSystem">
   <property name="text">
    <string>Load FileSystem</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionAdd_user">
   <property name="text">
    <string>Add user (only for ADMIN)</string>
   </property>
  </action>
  <action name="actionDelete_user">
   <property name="text">
    <string>Delete user (only for ADMIN)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
